### PR TITLE
OP-97 - New Anamnesis module

### PR DESCRIPTION
**Pull request by goto-eof**
See OP-97.

### CODE REVIEW

File: doc_user/UserManual.adoc [hunk 1]
```diff
@@ -1531,6 +1531,11 @@ Open Hospital attempts to automatically set the best quality for any webcam conn
 NOTE: If the camera does not function then change the flag to DEBUG in the configuration file to generate additional
 diagnostic information. Ask the Administrator or refer to the _Administrator’s Guide_.
 
+====== 8.4.6 anamnesis
+The medical history, case history, or anamnesis of a patient is information gained by a physician by asking specific questions, either to the patient or to other people who know the person and can give suitable information, with the aim of obtaining information useful in formulating a diagnosis and providing medical care to the patient.
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]: Suggested modified wording:
```suggestion
The medical history, case history, or anamnesis of a patient is information gained by a physician by asking specific questions either of the patient, or of other people who know the person and can give suitable information.  The aim is to obtain information useful in formulating a diagnosis and providing medical care to the patient.
```

File: doc_user/UserManual.adoc [hunk 2]
```diff
@@ -1531,6 +1531,11 @@ Open Hospital attempts to automatically set the best quality for any webcam conn
...
+
+To modify the anamnesis data of a patient select the patient in the *_Patient Browser_* window and then press the *[.underline]##E##dit* button. When the *_New Patient_* window is shown, then click on *_Anamnesis_* button, the record is available for changes. All the data can be changed.
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]: Correction and slightly changed wording:
```suggestion
To modify the anamnesis data of a patient, select the patient in the *_Patient Browser_* window and then press the *[.underline]##E##dit* button. When the *_Edit Patient_* window is shown, click on the *_Anamnesis_* button and the record is available for changes. All the data can be changed.
```

**Review by mwithi — CHANGES_REQUESTED** [review]
It seems that `angal.anamnesis.patient` (the title) is missing.

I would also add changed screenshot where the button has been introduced (Admission/Patient, New/Edit Patient, New/Edit OPD) with link to the new chapter

**Comment by mwithi** [comment]
Looks like the main screenshot has still the title missing.
For other screenshot where the "Anamnesis" button is missing, maybe we can just change the basic ones, like
- image14
- image78
- image85
- image92

wdyt @dbmalkovsky ?

**Comment by dbmalkovsky** [comment]
image247:

- As noted, the dialog title is missing
- Make the capitalization consistent:   `Past problems (Closed)` -> `Past Problems (Closed)` and `Recent problems (Open)` -> `Recent Problems (Open)`
- Change  `DrugsAddiction` to `Drug Addiction`
- It would be nice not to have to use `Nr` as an abbreviation; could the line be something like `Number [ ]  Delivered [ ] Aborted [ ]`?
- Should the titles of the boxes on the right all be plural to match the rest of the dialog?

image78:
There is no shortcut key for the `Anamnesis` button; how about using `m` and changing the `Merge` button shortcut key to `g`?

imsge88p:
There is no shortcut key for the `Anamnesis` button; see above.

As for adding the button
image14:  yes
image78: already has the button
image85: see image88p; the button is there just some screen shots haven't been updated
image92: yes

**Comment by goto-eof** [comment]
> 

done

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**